src/main/java/com/assignment/application/update/SalaryUpdate.java

```diff
@@ -53,19 +58,19 @@ public void setDepartmentName(String departmentName) {
         this.departmentName = departmentName;
     }
 
-    public long getCompanyId() {
+    public Long getCompanyId() {
         return companyId;
     }
 
-    public void setCompanyId(long companyId) {
+    public void setCompanyId(Long companyId) {
         this.companyId = companyId;
     }
 
-    public long getValue() {
+    public Long getValue() {
         return value;
     }
 
-    public void setValue(long value) {
+    public void setValue(Long value) {
         this.value = value;
     }
```

Comment:
Use @Data annotation instead of creating getters and setters for each field.

Reply:
It creates a problem for the mapping I am using because of id field. So in few files I need to declare getters and setters.